src/ressources/customers/CustomerEdit.tsx

```diff
@@ -0,0 +1,108 @@
+import { Box, Grid, Typography, Card } from "@mui/material";
+import {
+  DateInput,
+  Edit,
+  EditProps,
+  NullableBooleanInput,
+  PasswordInput,
+  SelectArrayInput,
+  SimpleForm,
+  TextInput,
+} from "react-admin";
+import { Aside } from "./Aside";
+
+export const CustomerEdit = (props: EditProps) => {
+  return (
+    <Edit
+      aside={<Aside />}
+      component={(props) => <Card variant="outlined" {...props} />}
+    >
+      <SimpleForm>
+        <Grid
+          container
+          spacing={3}
+          width={{ xs: "100%", lg: 1000 }}
+          textAlign="left"
+        >
+          <Grid item xs={12} md={8}>
+            <Typography variant="h6" gutterBottom>
+              Identity
+            </Typography>
+            <Box display={{ xs: "block", sm: "flex" }}>
+              <Box flex={1} mr={{ xs: 0, sm: "0.5em" }}>
+                <TextInput
+                  source="first_name"
+                  isRequired
+                  fullWidth
+                  color="secondary"
+                />
+              </Box>
+              <Box flex={1} ml={{ xs: 0, sm: "0.5em" }}>
+                <TextInput source="last_name" isRequired fullWidth />
+              </Box>
+            </Box>
+            <TextInput type="email" source="email" isRequired fullWidth />
+            <DateInput source="birthday" />
+            <Box m={1} />
```

Comment:
You probably don't need this one. All react-admin components should accept a `sx` prop as well